aws_eden_cli/utils.py

```diff
@@ -0,0 +1,107 @@
+import configparser
+import logging
+import os
+from pathlib import Path
+
+from . import consts
+
+logger = logging.getLogger()
+
+
+def read_config(path):
+    config = configparser.ConfigParser()
+    config.read(path)
+    return config
+
+
+def parse_config(args):
+    default_config_path = os.path.expanduser('~/.eden')
+    if not os.path.exists(default_config_path):
+        os.makedirs(default_config_path)
+
+    config_file_path = os.path.expanduser(args['config_path'])
+    config_file = Path(config_file_path)
+    if not config_file.is_file():
+        logger.info(f"Config file {config_file} is empty")
+        return None
+
+    return read_config(config_file)
+
+
+def config_write_overrides(config, args):
+    updated = False
+
+    profile_name = args['profile']
+
+    if profile_name not in config:
+        config[profile_name] = {}
+
+    for parameter in consts.parameters:
+        key = parameter['name']
```

Comment:
`parameter_name` is used for variable name in `create_envvar_dict` function. I prefer that one.

Reply:
This one changes on every iteration as well